ui/command_buffer_model.h

```diff
@@ -70,4 +70,5 @@ public slots:
     std::map<uint64_t, QModelIndex> m_node_index_to_parent_map;
     const Dive::CommandHierarchy   &m_command_hierarchy;
     const Dive::Topology           *m_topology_ptr = nullptr;
+    bool                            m_show_level_column;
```

Comment:
better to be initialized with a default value.

Reply:
> better to be initialized with a default value.

Done